src/utils/DockerUtil.js

```diff
@@ -263,6 +271,68 @@ var DockerUtil = {
     });
   },
 
+  fetchAllNetworks () {
+    this.client.listNetworks((err, networks) => {
+      if (err) {
+        networkActions.error(err)
+      } else {
+        networks = networks.filter(n => n.Name !== 'host').sort((n1, n2) => {
```

Comment:
By default the container will be attached to the bridge network. You cannot have both the host and some other network, but can create a container with the host network:
```
docker run --net host --rm -it alpine sh
```